src/components/NodeViewBase.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import NodeView from './NodeView'
+import {
+  RudolfStore,
+  CustomDispatch,
+  updateJustification,
+} from '../RudolfReducer'
+import nodeviewJSS, { rowHeight } from '../styles/NodeView_styles'
+import { range } from '../util/helpers'
+import AutosizingInput from './AutosizingInput'
+
+type Props = {
+  currentState: RudolfStore
+  dispatch: CustomDispatch
+}
+
+const NodeViewBase = ({ currentState, dispatch }: Props) => {
```

Comment:
This could be renamed to `TruthTree`, since it handles more than just the root node.